src/remote.js

```diff
@@ -259,20 +278,20 @@
                       util.err(error);
                     }
                   )
-                },
-                function(error) {
-                  util.err(error);
                 }
-              )
+              },
+              function(error) {
+                util.err("Unable to build this project. " + error);
+              }
+            ),
+            function(error) {
+              util.err('Unable to create monaca project: ' + JSON.stringify(error));
             }
-          },
-          function(err) {
-            util.err("Unable to build this project " + err);
           }
-        )
-      },
-      function(err) {
-        util.err(err);
+        ),
+        function(err) {
+          util.err(err);
```

Comment:
Please see what's the type of `err` object and confirm the user gets appropriate error message.